tests/ui_tests/test_admin_page.py

```diff
@@ -53,6 +73,37 @@ def settings_page(app_fs: ADCMTest) -> AdminSettingsPage:
     return AdminSettingsPage(app_fs.driver, app_fs.adcm.url).open()
 
 
+@allure.step("Upload cluster bundle")
+def cluster_bundle(sdk_client_fs: ADCMClient, data_dir_name: str) -> Bundle:
+    """Upload cluster bundle"""
+
+    return sdk_client_fs.upload_from_fs(os.path.join(utils.get_data_dir(__file__), data_dir_name))
+
+
+@pytest.fixture()
+def create_cluster_with_service(sdk_client_fs: ADCMClient) -> Tuple[Cluster, Service]:
+    """Create cluster and add service"""
+
+    bundle = cluster_bundle(sdk_client_fs, BUNDLE)
+    cluster = bundle.cluster_create(name=CLUSTER_NAME)
+    return cluster, cluster.service_add(name=SERVICE_NAME)
+
+
+@pytest.fixture()
+def create_cluster_with_component(
+    create_cluster_with_service: Tuple[Cluster, Service], sdk_client_fs: ADCMClient
+) -> Tuple[Cluster, Service, Host, Provider]:
+    """Create cluster with component"""
+
+    cluster, service = create_cluster_with_service
+    provider_bundle = sdk_client_fs.upload_from_fs(os.path.join(utils.get_data_dir(__file__), 'provider'))
+    provider = provider_bundle.provider_create('test_provider')
+    host = provider.host_create('test-host')
+    cluster.host_add(host)
+    cluster.hostcomponent_set((host, service.component(name=FIRST_COMPONENT_NAME)))
+    return cluster, service, host, provider
```

Comment:
For further notice: I suggest keeping one order in return objects.
Cluster -> Service -> Component -> Provider -> Host
That way it'll be easier to read similar code pieces